mapie/tests/test_classification.py

```diff
@@ -267,21 +332,25 @@
 }
 
 COVERAGES = {
-    "score": 6 / 9,
-    "score_cv_mean": 1,
-    "score_cv_crossval": 1,
-    "cumulated_score_include": 1,
-    "cumulated_score_not_include": 5 / 9,
-    "cumulated_score_randomized": 6 / 9,
-    "cumulated_score_include_cv_mean": 1,
-    "cumulated_score_not_include_cv_mean": 5 / 9,
-    "cumulated_score_randomized_cv_mean": 5 / 9,
-    "cumulated_score_include_cv_crossval": 2 / 9,
-    "cumulated_score_not_include_cv_crossval": 0,
-    "cumulated_score_randomized_cv_crossval": 4 / 9,
-    "naive": 5 / 9,
-    "top_k": 1,
-    "raps": 1,
+    "score": 6/9,
+    "score_split": 8/9,
+    "score_cv_mean": 1.0,
+    "score_cv_crossval": 1.0,
+    "cumulated_score_include": 1.0,
+    "cumulated_score_not_include": 5/9,
+    "cumulated_score_randomized": 6/9,
+    "cumulated_score_include_split": 8/9,
+    "cumulated_score_not_include_split": 5/9,
+    "cumulated_score_randomized_split": 7/9,
+    "cumulated_score_include_cv_mean": 1.0,
+    "cumulated_score_not_include_cv_mean": 5/9,
+    "cumulated_score_randomized_cv_mean": 8/9,
+    "cumulated_score_include_cv_crossval": 4/9,
+    "cumulated_score_not_include_cv_crossval": 1/9,
+    "cumulated_score_randomized_cv_crossval": 7/9,
+    "naive": 5/9,
```

Comment:
add split for top_k, raps, and randomized_raps

Reply:
Done.

Reply:
still missing for RAPS I think

Reply:
I did not implement the split method for RAPS.

Reply:
Done now.